### PR TITLE
a mini theme for blockly

### PR DESCRIPTION
**Pull request by SallyMcGrath**
There's a whole themeing system but the grey block was bugging me so here's a super fast override to theme it in with our variables.

I stacked this PR on my last branch as it's dependent on the previous code

### CODE REVIEW

**Comment by SallyMcGrath** [comment]
Note: I moved the base of this PR to main as the previous dependent PR was merged.

File: style.css [hunk 1]
```diff
@@ -231,6 +223,39 @@ footer {
...
+.blocklyToolboxDiv {
+  border: var(--theme-border);
+  padding: 20px 10px;
+  box-shadow: 2px 6px 0 var(--theme-color--pop);
+}
+.blocklyMainBackground {
+  fill: transparent !important;
+  border: var(--theme-border);
+  stroke: var(--theme-color--accent);
+  stroke-width: 0;
```
Comment (gregdyke) [inline style.css:247]:
```suggestion
```
I think showing where the boundary of the box is makes it a bit more crowded, but more obvious that there is a box where things can be placed? Alternatively, play with the transparency>

**Review by gregdyke — CHANGES_REQUESTED** [review]
Wow! (also I found the "toggle night mode" button in my developer tools, so I got to see day mode :D)

Thanks for the flash. Good choice of implementation I think!!

**Comment by SallyMcGrath** [comment]
👋 For those following along at home, you can switch your themes in Devtools by pulling up the command palette:

**Shift Command P** 

Then type `emu` to bring up your emulation options. Choose prefers colour scheme light/dark.  / 🧇

**Review by gregdyke — APPROVED** [review]

**Pull request merged**